src/components/molecules/HomeArrowContainer.vue

```diff
@@ -1,13 +1,14 @@
 <template>
 	<div class="home__arrow-container">
-		<homeArrow v-bind:arrow="arrow" v-bind:key="arrow.id" v-for="arrow in arrows"/>
+		<homeArrow v-bind:containers="containers" v-bind:arrow="arrow" v-bind:counter="counter" v-bind:key="arrow.id" v-for="arrow in arrows"/>
```

Comment:
why you need to put `for` loop and create 10 lines data object to render 2 arrows?